### PR TITLE
fix: Update markdown hyperlink regex

### PR DESCRIPTION
**Pull request by BapRx**
Hey, I had the same issue as in #1162 and I noticed that the fix was revert. Here's my PR to fix the issue.

Before:

After:
image

### CODE REVIEW

File: web/src/labs/marked/parser/Link.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import BoldEmphasis from "./BoldEmphasis";
 import PlainText from "./PlainText";
 import { matcher } from "../matcher";
 
-export const LINK_REG = /\[(.*?)\]\((.+?)\)+/;
+export const LINK_REG = /\[([^\]]+)\]\(([^)]+)\)/;
```
Comment (boojack) [inline web/src/labs/marked/parser/Link.tsx:9]:
```suggestion
export const LINK_REG = /\[(.*?)\]\(([^\s]+)\)/;
```

Comment (BapRx) [reply web/src/labs/marked/parser/Link.tsx:9]: I don't understand your update, your suggestion doesn't work:
image

Comment (boojack) [reply web/src/labs/marked/parser/Link.tsx:9]: I see

**Review by boojack — APPROVED** [review]
LGTM, thanks for your contribution!

**Pull request merged**